modules/docs/src/main/mdoc/docs/memoization.md

```diff
@@ -99,7 +99,7 @@ class Bar(a: Int) {
 then you want the cache key to depend on the values of both `a` and `b`. In that case, you need to use a different implementation of [MethodCallToStringConverter](https://github.com/cb372/scalacache/blob/master/modules/core/shared/src/main/scala/scalacache/memoization/MethodCallToStringConverter.scala), like this:
 
 ```scala mdoc:silent
-implicit val cacheConfig = CacheConfig(
+implicit lazy val cacheConfig: CacheConfig = CacheConfig(
```

Comment:
Seems like the `implicit val catsCache: Cache[IO, Cat] = MemcachedCache("localhost:11211")` earlier on in this file constructs a forward reference to this implicit val (presumably it didn't before the tut->mdoc change); so this needs to be lazy or there's a weird NPE/AbstractMethodError (depending on scala version) thrown when running docs/mdoc

Reply:
Looks like the rest works without this snippet, so I'd rather remove the lazy but exclude this snippet from mdoc.